packages/components/src/internal/components/forms/input/SampleStatusInput.tsx

```diff
@@ -26,6 +26,7 @@ interface SampleStatusInputProps extends Omit<QuerySelectOwnProps, 'schemaQuery'
 
 export const SampleStatusInput: FC<SampleStatusInputProps> = memo(props => {
     const { api, col, onAdditionalFormDataChange, renderLabelField, ...selectInputProps } = props;
+    delete selectInputProps['containerFilter']; // use col.lookup.containerFilter instead
```

Comment:
nit: Why `selectInputProps['containerFilter']` instead of `selectInputProps.containerFilter`?